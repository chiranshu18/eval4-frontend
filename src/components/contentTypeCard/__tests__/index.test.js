import ContentTypeCard from '../index';
import { render , screen} from '@testing-library/react';
import React from 'react';

describe('Content Types Card', () => {
  it('Should render correctly', () => {
    const {asFragment} = render(<ContentTypeCard />);
    expect(asFragment()).toMatchSnapshot();
  });
});