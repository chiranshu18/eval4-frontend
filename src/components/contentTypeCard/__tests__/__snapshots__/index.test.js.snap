// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content Types Card Should render correctly 1`] = `
<DocumentFragment>
  <div
    class="content-type-card"
  >
    <p
      class="add-new-content-type-card-text"
    />
    <p
      class="add-new-content-type-card-text"
    />
  </div>
</DocumentFragment>
`;
